parser/ParseStatement.hpp

```diff
@@ -153,6 +156,7 @@ class ParseStatementCreateTable : public ParseStatement {
  private:
   std::unique_ptr<ParseString> relation_name_;
   std::unique_ptr<PtrList<ParseAttributeDefinition> > attribute_definition_list_;
+  std::unique_ptr<ParseBlockProperties> opt_block_properties_;
```

Comment:
You should probably add a const accessor method so that this field can be read after the object is created.